serving-tiles/manually-building-a-tile-server-20-04-lts.md

```diff
@@ -85,6 +85,13 @@ Next we'll install osm2pgsql:
 
     sudo apt install osm2pgsql
 
+## Postgresql optimization
+
+If you have a default installation of postgresql it is very unlikely that it will be configured to offer good performance by default.
```

Comment:
```suggestion
If you have a default installation of PostgreSQL it is very unlikely that it will be configured to offer good performance by default.
```